neural_compressor/strategy/st_utils/tuning_space.py

```diff
@@ -166,8 +166,15 @@ def _merge_op_cfg(self, op_cap, op_user_cfg, fw_op_cap):
                         op_cap['quant'][quant_mode_flag] = deepcopy(fw_quant_cap)
                     for item_name, item_options in op_user_cfg[key].items():
                         if item_options is not None and key in fw_quant_cap and item_name in fw_quant_cap[key]:
-                            merged_options = [option for option in item_options if \
-                                              option in fw_quant_cap[key][item_name]]
+                            merged_options = []
+                            for option in item_options:
+                                if option in fw_quant_cap[key][item_name]:
+                                    merged_options.append(option)
+                                else:
+                                    logger.warning("By default, {1}: {2} is not supported for {0} ".format(

```

Comment:
doesn't it mean our UT coverage is not enough? pls @ liuyi to fix

Reply:
I have enhanced the UT. 
@yiliu30, please have a check.

Reply:
Hi @ftian1, I think @xin3he wants to add this warning information to guide users to update the frameworkname_xyz.yaml if they're going to use the quantization config out of frameworkname_xyz.yaml. 
For example, users want to use `per-tensor` quantization for  `activation` for the PyTorch model. 
This doesn't mean we don't have enough UT coverage.